include/grub/slr_table.h

```diff
@@ -285,6 +313,9 @@ grub_slr_init_table(struct grub_slr_table *slrt, grub_uint16_t architecture,
 {
   struct grub_slr_entry_hdr *end;
 
+  if (max_size < sizeof(*slrt) + sizeof(*end))
+    grub_fatal ("Requested SLRT max size (%d B) is too small\n", max_size);
```

Comment:
```suggestion
    grub_fatal ("Requested SLRT max size (%u B) is too small\n", max_size);
```

or use the appropriate `GRUB_PRIu32` macro.